### DISCUSSION TITLE
Do we remove listeners when using the socket.once method

### DISCUSSION THREAD

**Discussion by ElmcHostedApplications** [socketio/socket.io#4778] [General]
If I create a listener using the once method do I still need to remove the listener or is the listener automatically removed. I just want to make sure I don't get to many listener in the react native project

```
socket.once("eventName" (data) =>{
 // do something
})
```

Do we need to use ```socket.off```

**Comment by darrachequesne** [comment]
Hi! No, calling `socket.off()` is not needed, the listener is automatically removed.
